easybuild/easyconfigs/m/Mesa/Mesa-20.0.2-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,80 @@
+# This is a Mesa using software rendering via Gallium-DRI and libglvnd
+#
+# - libglvnd can dynamically choose between system-installed NVidia
+#   libGLX/libEGL or the software renderers provided by this Mesa
+# - EGL is available
+#
+# Software renderers are enabled based on system architecture:
+# - llvmpipe: the high-performance Gallium LLVM driver
+# - swr: Intel's OpenSWR
+# Default renderer is llvmpipe. SWR can be enabled by setting the environment
+# variable GALLIUM_DRIVER=swr
+
+name = 'Mesa'
+version = '20.0.2'
+
+homepage = 'https://www.mesa3d.org/'
+description = """Mesa is an open-source implementation of the OpenGL specification -
+ a system for rendering interactive 3D graphics."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = [
+    'https://mesa.freedesktop.org/archive/',
+    'https://mesa.freedesktop.org/archive/%(version)s',
+    'ftp://ftp.freedesktop.org/pub/mesa/%(version)s',
+    'ftp://ftp.freedesktop.org/pub/mesa/older-versions/%(version_major)s.x/%(version)s',
+    'ftp://ftp.freedesktop.org/pub/mesa/older-versions/%(version_major)s.x',
+]
+sources = [SOURCELOWER_TAR_XZ]
+patches = ['Mesa-%(version)s-match-swrast-loosely.patch']
+checksums = [
+    'aa54f1cb669550606aab8ceb475105d15aeb814fca5a778ce70d0fd10e98e86f',  # mesa-20.0.2.tar.xz
+    'e86abc2b9a9ad3e2fc604ad5d9bfe15a62e842e9900365fe7061849ea7438d90',  # Mesa-20.0.2-match-swrast-loosely.patch
+]
+
+builddependencies = [
+    ('binutils', '2.34'),
+    ('Meson', '0.53.2', '-Python-3.8.2'),
+    ('Ninja', '1.10.0'),
+    ('flex', '2.6.4'),
+    ('Bison', '3.5.3'),
+    ('pkg-config', '0.29.2'),
+    ('Mako', '1.1.2'),
+    ('libxml2', '2.9.10'),
+    ('expat', '2.2.9'),
+    ('gettext', '0.20.1'),
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('zstd', '1.4.4'),
+    ('libdrm', '2.4.100'),
+    ('libglvnd', '1.2.0'),
+    ('libunwind', '1.3.1'),
+    ('LLVM', '9.0.1'),
+    ('X11', '20200222'),
+]
+
+configopts = " -Dplatforms=x11 -Dosmesa=gallium -Ddri-drivers='' -Dvulkan-drivers=''"
+configopts += " -Dllvm=true -Dshared-llvm=true"
+configopts += " -Dlibunwind=true"
+configopts += " -Dglvnd=true"
+
+# symlink indirect to mesa GLX, similar to Debian, see
+# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=881789
+# This helps in certain X forwarding situations (e.g. XQuartz)
+postinstallcmds = ["ln -s libGLX_mesa.so.0 %(installdir)s/lib/libGLX_indirect.so.0"]
+
+# Tells libglvnd where to find EGL libraries
+modextrapaths = {"__EGL_VENDOR_LIBRARY_DIRS": "share/glvnd/egl_vendor.d"}
+
+sanity_check_paths = {
```

Comment:
if https://github.com/easybuilders/easybuild-easyblocks/pull/1892 is enhanced with the changes proposed in https://github.com/bear-rsg/easybuild-easyblocks/pull/17, this custom `sanity_check_paths` should be stripped out

Reply:
The sanity checks in your PR will fail with this version of Mesa, see my comment at https://github.com/bear-rsg/easybuild-easyblocks/pull/17#discussion_r396122428

Reply:
fixed now, see https://github.com/easybuilders/easybuild-easyblocks/pull/1892